utils/src/main/java/by/khadasevich/utils/StringUtils.java

```diff
@@ -0,0 +1,15 @@
+package by.khadasevich.utils;
+
+import static org.apache.commons.lang3.math.NumberUtils.*;
+
+public class StringUtils {
+    public static boolean isPositiveNumber(String str) {
+        if (isCreatable(str)) {
```

Comment:
here if number is not creatable from string will show you that number is negative. that way please remove that string and allow java to throw corresponding exception for that. also please write a test for that case

Reply:
I believe in task they assume NumberUtils.isDigits to be used